sdk/scheduler/src/test/resources/invalid-pod-name.yml

```diff
@@ -6,7 +6,7 @@ pods:
     resource-sets:
       meta-data-resources:
         cpus: 0.1
-        mem: 512
+        memory: 512
```

Comment:
(These misspelled fields were being missed in the tests -- they were throwing a parse exception but not the one the tests were meant for)